docs/Lama2/docs/reference/architecture.md

```diff
@@ -48,7 +48,7 @@ From a high level, how does it work now?
 	2. Else if block is Requestor block
 		1. Replace variables with values in the following order
 			1. Try fetch variable from Javascript VM
-			2. If (1) fails, try fetch variable from `l2.env`
+			2. If (1) fails, try fetch variable from `l2config.env` and `l2.env`
```

Comment:
"overwriting" is an implementation detail. When you talk to users - how does it work?

1. JS VM variable
2. Local env variable
3. l2config env variable.

Always explain in this way. The implementation detail doesn't matter at documentation/explanation level.

Reply:
Okay got it